packages/components/addon/components/hds/pagination/nav/ellipsis.hbs

```diff
@@ -1,3 +1 @@
-<li class="hds-pagination-nav__page-item" ...attributes>
-  <div class="hds-pagination-nav__ellipsis">...</div>
-</li>
\ No newline at end of file
+<div class="hds-pagination-nav__ellipsis" ...attributes>...</div>
```

Comment:
Can you remind me how this would be handled for assistive tech?

Reply:
@MelSumner looking at other design systems, this is exactly how it's implemented in all of them:
- https://design-system.service.gov.uk/components/pagination/
- https://design-system.w3.org/components/pagination.html
- https://atlassian.design/components/pagination/examples
- https://ons-design-system.netlify.app/components/pagination/
- https://primer.style/css/components/pagination

What would you suggest to do?